Quantum-Control-Applications/ESR/Use case 1 - Sekhar Lab - CPMG/README.md

```diff
@@ -0,0 +1,102 @@
+# Carr-Purcell-Meiboom-Gill (CPMG) in an NV ensemble with electron spin resonance (ESR)
```

Comment:
This only works in an NV-center experiment right? If so I would put all the codes from ESR under NV-center which makes more sense, what do you think?